src/plonk/permutation.rs

```diff
@@ -88,7 +88,7 @@ pub struct VerifyingKey<F: PrimeField, CS: PolynomialCommitmentScheme<F>> {
 }
 
 impl<F: PrimeField, CS: PolynomialCommitmentScheme<F>> VerifyingKey<F, CS> {
-    /// Returns commitments of sigma polynomials
+    /// Returns the commitments of the verifying key.
```

Comment:
Hum. The "sigma poly" was not that bad of a comment.

```suggestion
    /// Returns the (permutation argument) commitments of the verifying key.
```